Client/Frontend/Browser/BrowserToolbar.swift

```diff
@@ -11,27 +11,49 @@ protocol BrowserToolbarDelegate: class {
     func browserToolbarDidPressForward(browserToolbar: BrowserToolbar, button: UIButton)
     func browserToolbarDidLongPressBack(browserToolbar: BrowserToolbar, button: UIButton)
     func browserToolbarDidLongPressForward(browserToolbar: BrowserToolbar, button: UIButton)
+    func browserToolbarDidPressReload(browserToolbar: BrowserToolbar, button: UIButton)
+    func browserToolbarDidPressStop(browserToolbar: BrowserToolbar, button: UIButton)
     func browserToolbarDidPressBookmark(browserToolbar: BrowserToolbar, button: UIButton)
     func browserToolbarDidLongPressBookmark(browserToolbar: BrowserToolbar, button: UIButton)
     func browserToolbarDidPressShare(browserToolbar: BrowserToolbar, button: UIButton)
 }
 
 private let ButtonInset = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+private let NavButtonInset = UIEdgeInsets(top: 12, left: 10, bottom: 12, right: 10)
 
 class BrowserToolbar: Toolbar {
     weak var browserToolbarDelegate: BrowserToolbarDelegate?
 
+    private let ImageReload = UIImage(named: "reload")
+    private let ImageReloadPressed = UIImage(named: "reloadPressed")
+    private let ImageStop = UIImage(named: "stop")
+    private let ImageStopPressed = UIImage(named: "stopPressed")
+
     private let shareButton: UIButton
     private let bookmarkButton: UIButton
     private let forwardButton: UIButton
     private let backButton: UIButton
+    private let stopReloadButton: UIButton
     private let longPressGestureBackButton: UILongPressGestureRecognizer!
     private let longPressGestureForwardButton: UILongPressGestureRecognizer!
     private let longPressGestureBookmarkButton: UILongPressGestureRecognizer!
 
+    var loading: Bool = false {
+        didSet {
+            if loading {
+                stopReloadButton.setImage(ImageStop, forState: .Normal)
+                stopReloadButton.setImage(ImageStopPressed, forState: .Highlighted)
+            } else {
+                stopReloadButton.setImage(ImageReload, forState: .Normal)
+                stopReloadButton.setImage(ImageReloadPressed, forState: .Highlighted)
+            }
+        }
+    }
```

Comment:
Would you mind also setting `stopReloadButton.accessibilityLabel` and `stopReloadButton.accessibilityHint` here, based on `loading` status?